change/@ni-nimble-components-ffc88008-f921-4cf2-aa10-dd2e6c97f3e8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@leslieab saw some recent pushes, just a reminder when ready for a re-review to hit the reset reviewer button for those who have not approved.

If the changes were significant enough that you think current approved reviewers should be reset to take another look you could chose to reset those reviewers for a re-review as well.

![image](https://github.com/ni/nimble/assets/1588923/5532cdb5-3209-442f-a0b2-5301d43ee477)

Reply:
Ooh, I didn't even know that existed. I will do that, thanks!